QualificationChecker/Controllers/OrganizationController.cs

```diff
@@ -0,0 +1,51 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using QualificationChecker.Models;
+
+namespace QualificationChecker.Controllers
+{
+    [ApiController]
+    [Route("api/qualifiedcandidates")]
+    public partial class OrganizationController : ControllerBase
+    {
+        [HttpPost("showQualified/")]
+        public IActionResult ShowQualifiedCandidates([FromBody] MatchRequest request)
+        {
+            if (request.Candidates == null)
+            {
+                return Ok("No Candidate at this time");
+            }
+
+            if (request.Organization?.OrgQuestions == null)
+            {
+                return Ok("Your organization does not have any Qualification questions; please add them.");
+            }
+
+            try
+            {
+                List<Candidate> qualifiedCandidates = new List<Candidate>();
+
+                foreach (var candidate in request.Candidates)
+                {
+                    var myqualifiedCandidates = new CandidateQualificationEvaluator(candidate, request.Organization);
+                    myqualifiedCandidates.FindQualifiedCandidates(candidate);
+                    qualifiedCandidates.AddRange(myqualifiedCandidates.ReturnListOfCandidate());
+                }
+
+                if (qualifiedCandidates.Count >= 1)
+                {
+                    return Ok(qualifiedCandidates);
+                }
+                else
+                {
+                    return Ok("No Matching Candidates");
+                }
+            }
+            catch (Exception ex)
+            {
+                return StatusCode(500, $"{ex} error occurred while processing the request.");
+            }
+        }
+
+    }
+}
```

Comment:
Much cleaner!  You can actually ignore the try/catch, and setup asp.net to return a custom response for any exception ... simplifying your code. The only reason to try/catch is if you want to DO something with the exception